<ui version="4.0" >
 <class>PromedioBase</class>
 <widget class="QWidget" name="PromedioBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Compras</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="mui_undcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>Und. Compradas</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" >
       <widget class="QLineEdit" name="mui_undavgcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Cantidad Media</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="mui_undmayorcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Cantidad mayor</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLineEdit" name="mui_undmenorcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Cantidad Menor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Total</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Precio Medio</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="mui_avgcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="mui_menorcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Precio Mayor</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Precio Menor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="mui_totalcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="mui_mayorcompras" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Ventas</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_16" >
        <property name="text" >
         <string>Und. Vendidas</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Cantidad mayor</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="mui_undventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="mui_avgventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Precio Medio</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="mui_mayorventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="mui_undmayorventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLineEdit" name="mui_undmenorventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1" >
       <widget class="QLineEdit" name="mui_undavgventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="mui_totalventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Precio Menor</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Cantidad Menor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Precio Mayor</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="mui_menorventas" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Total</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>Cantidad Media</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
